docker/app/kafkahq

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env sh
 
 # Read user-defined JVM options from jvm.options file
-JVM_OPTS_FILE=${JVM_OPTS_FILE:-/app/jvm.options}
+if [ -z "${JVM_OPTS_FILE}" ]; then
```

Comment:
```
# JVM_OPTS_FILE=${JVM_OPTS_FILE:-/app/jvm.options}
# echo $JVM_OPTS_FILE
/app/jvm.options
# JVM_OPTS_FILE=${JVM_OPTS_FILE:-/app/jvm.options}^C
# export JVM_OPTS_FILE="/tmp/bla"
# JVM_OPTS_FILE=${JVM_OPTS_FILE:-/app/jvm.options}
# echo $JVM_OPTS_FILE
/tmp/bla
```

This `if` is exactly the same than the previous one `${JVM_OPTS_FILE:-/app/jvm.options}` that will take the default parameters only if `JVM_OPTS_FILE` is not defined